README.md

```diff
@@ -1,21 +1,25 @@
-This is a small windows script to help extract Wings! amiga disks from the GOG release.
+This is a small Windows script to help extract Wings! amiga disks and the Kickstart 1.2 firmware from the GOG release. Also possible to extract from Linux, as explained below.
 
 # Requirements
 
-7z should be in your path, I got this on Windows by installing 7zip
+To have [7-Zip](https://www.7-zip.org/) installed.
 
 # Instructions
 
-1. Install Wings!
-2. find the Wings.exe, typically in "C:\GOG Games\Wings! Classic"
-3. drag and drop it over the extractwings.bat file
-4. you should now have two .adf files wings-1.adf and wings-2.adf these are the game disks
+1. Open the command prompt and run the command `set PATH=%PATH%;C:\Program Files\7-Zip\` (change `C:\Program Files\7-Zip\` to where you installed 7-Zip, if you didn't install it in the default folder)
+2. Download [extractwings's latest build](https://github.com/sigboe/extractwings/archive/refs/heads/master.zip)
```

Comment:
I like these instruction changes, but do you think we should put it as a release instead?

Reply:
A release sounds interesting too.